frontend/src/metabase/components/FieldValuesWidget.jsx

```diff
@@ -446,17 +454,30 @@ function shouldList(fields, disableSearch) {
   );
 }
 
-function getNonSearchableTokenFieldPlaceholder(firstField) {
-  if (firstField.isID()) {
-    return t`Enter an ID`;
-  } else if (firstField.isString()) {
+function getNonSearchableTokenFieldPlaceholder(firstField, parameter) {
+  if (parameter) {
```

Comment:
I despise nested if, but I just want to keep things simple for now to verify if the approach is valid first.